fitsio/fitslib.py

```diff
@@ -1800,6 +1845,26 @@ def get_qmethod(qmethod):
     return _qmethod_map[qmethod]
 
 
+def get_dither_seed(dither_seed):
+    if dither_seed is None or dither_seed in ['clock', 'CLOCK']:
+        # 0 means do not set the seed, use the system clock. This is the
+        # default
+        seed_out = 0
+    elif dither_seed in ['checksum', 'CHECKSUM']:
+        seed_out = -1
+    else:
+        # must fit in an int
+        seed_out = numpy.int32(dither_seed)
+
+        if seed_out < -1 or seed_out > 10_000:
```

Comment:
```suggestion
        if seed_out > 10_000:
```

Any negative value is allowed by the fits routine. Fun!